### PR TITLE
add lib dir to Libdl.DL_LOAD_PATH

### PR DESCRIPTION
**Pull request by larssp**
resolves build failure #63

### CODE REVIEW

**Comment by larssp** [comment]
The MathOptInterface v0.5 deprecation of cantransform, canaddconstraint, canaddvariable, canset, canmodify, and candelete functions has to be resolved next

File: REQUIRE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 julia 0.6
 BinDeps
 MathProgBase 0.5.5 0.8
-MathOptInterface 0.4.1 0.5
+MathOptInterface 0.4.1 0.6
```
Comment (larssp) [inline REQUIRE]: JuliaPackaging/WinRPM.jl#158

**Comment by mlubin** [comment]
Before you go too far down this path, please put changes for updating from MOI 0.5 to 0.6 in a separate PR to keep things easy to review. Also keep in mind that tests need to pass on both Julia 0.6 and 0.7 before we merge. If you can get them passing on 1.0 as well, that's even better.

File: src/CbcSolverInterface.jl [hunk 2]
```diff
@@ -2,7 +2,7 @@ module CbcMathProgSolverInterface
 
 using Cbc.CbcCInterface
 
-importall MathProgBase.SolverInterface
+using MathProgBase.SolverInterface
```
Comment (GunnarFarneback) [inline src/CbcSolverInterface.jl]: `using` only replaces parts of what `importall` was doing, but see #65.

**Comment by larssp** [comment]
this PR is premature, closing it.

**Pull request closed without merging**